backend/src/openarchiefbeheer/destruction/tests/e2e/features/test_feature_list_select_all.py

```diff
@@ -0,0 +1,76 @@
+from openarchiefbeheer.utils.tests.gherkin import GherkinLikeTestCase
+
+
+@sync_to_async()
+def get_list(name):
+    return DestructionList.objects.get(name=name)
+
+
+@tag("e2e")
+class FeatureListCreateSelectAllTests(GherkinLikeTestCase):
+    async def test_scenario_record_manager_creates_list_with_select_all(self):
+        async with browser_page() as page:
+            await self.given.record_manager_exists()
+            await self.given.reviewer_exists(username="Beoordelaar 1")
+            await self.given.reviewer_exists(username="Beoordelaar 2")
+            await self.given.zaken_are_indexed(200)
+
+            await self.when.record_manager_logs_in(page)
+            await self.then.path_should_be(page, "/destruction-lists")
+
+            await self.when.user_clicks_button(page, "Vernietigingslijst opstellen")
+            await self.then.path_should_be(page, "/destruction-lists/create")
+
+            await self.when.user_clicks_checkbox(page, "(de)selecteer 200 pagina's", index=0)
+            await self.when.user_clicks_button(page, "Vernietigingslijst opstellen", index=1)
+            await self.when.user_fills_form_field(page, "Naam", "Destruction list select all")
+            await self.when.user_fills_form_field(page, "Eerste reviewer", "Beoordelaar 1")
+            await self.when.user_fills_form_field(page, "Tweede reviewer", "Beoordelaar 2")
+            await self.when.user_clicks_button(page, "Verzenden")
+
+            await self.then.path_should_be(page, "/destruction-lists")
+            await self.then.page_should_contain_text(page, "Destruction list select all")
+
+            list = await get_list("Destruction list select all")
+
+            await self.then.list_should_have_number_of_items(list, 200)
+
+    async def test_scenario_record_manager_creates_list_with_select_all_and_filters(self):
+        async with browser_page() as page:
+            await self.given.record_manager_exists()
+            await self.given.reviewer_exists(username="Beoordelaar 1")
+            await self.given.reviewer_exists(username="Beoordelaar 2")
+            # This creates a zaak that should NOT be added to the list (since it's already in a list)
+            await self.given.list_item_exists()
+            # These zaken should be on the create page
+            await self.given.zaken_are_indexed(5, omschrijving="Test 1")
+            await self.given.zaken_are_indexed(5, omschrijving="Test 2")
+            
+            await self.when.record_manager_logs_in(page)
+            await self.then.path_should_be(page, "/destruction-lists")
+
+            await self.when.user_clicks_button(page, "Vernietigingslijst opstellen")
+            await self.then.path_should_be(page, "/destruction-lists/create")
+
+            await self.when.user_filters_zaken(page, "omschrijving", "Test 1")
+            await self.when.user_clicks_checkbox(page, "(de)selecteer 5 pagina's", index=0)
```

Comment:
Note: this should be changed when https://github.com/maykinmedia/admin-ui/issues/126 is fixed